pkgs/by-name/de/deadlock-modmanager/package.nix

```
@@ -0,0 +1,155 @@
+{
+  lib,
+  fetchFromGitHub,
+  rustPlatform,
+  cargo-tauri,
+  nodejs,
+  pnpm_9,
+  pkg-config,
+  wrapGAppsHook3,
+  desktop-file-utils,
+  webkitgtk_4_1,
+  cairo,
+  gdk-pixbuf,
+  glib,
+  glib-networking,
+  gtk3,
+  libsoup_3,
+  pango,
+  openssl,
+  bzip2,
+  gst_all_1,
+  makeDesktopItem,
+  fontconfig,
+  nix-update-script,
+}:
+
+rustPlatform.buildRustPackage rec {
```

```suggestion
rustPlatform.buildRustPackage (finalAttrs: {
```

This is the newer style, not currently enforced, so normally this would be a nitpick; but in this case it makes overriding massively easier due to us using `cargoRoot` and all the stuff that's used to handle the pnpm deps.